app/src/pages/inside/common/launchFiltersToolbar/filterList/filterItem/filterTooltipIcon/filterTooltipIcon.jsx

```diff
@@ -0,0 +1,18 @@
+import classNames from 'classnames/bind';
+import Parser from 'html-react-parser';
+import { ALIGN_LEFT } from 'components/main/tooltips/constants';
+import { withHoverableTooltip } from 'components/main/tooltips/hoverableTooltip';
+import { TextTooltip } from 'components/main/tooltips/textTooltip';
+import TooltipIcon from 'common/img/tooltip-icon-inline.svg';
+import styles from './filterTooltipIcon.scss';
+
+const cx = classNames.bind(styles);
+
+export const FilterTooltipIcon = withHoverableTooltip({
```

Comment:
Should this tooltip be hoverable? (it isn't hoverable on the fourth version of RP)

Reply:
I think it shouldn't be hoverable. Fixed